Automattic/wp-calypso
# Add 'Unlimited sites (Akismet Business)' as option in Ak Pro quantity dropdown in checkout

This PR adds "Unlimited sites (Akismet Business)" as an option in the dropdown, that when “Unlimited sites (Akismet Business)” is selected in the dropdown, we replace the Pro product in the cart with Akismet Business.  and then If 1 - 4 is selected in the dropdown, replace the Business product with Akismet Pro.



**Related to:**
- PT: Akismet Checkout – Multiple licenses: pbNhbs-8My-p2
- Akismet Checkout: Multiple Licenses – Implementation Plan: pbNhbs-8P6-p2


## Testing Instructions

- First apply patch D128326-code to your sandbox and sandbox `public-api.wordpress.com`.
- Checkout this PR branch and `yarn-start`
- Go to https://akismet.com/pricing and select/click the  "Get Pro" button. 
- On the checkout page, in the URL, replace `https://wordpress.com` with `http://calypso.localhost:3000` and press enter.
- You should see the "Number of licenses" dropdown, like in the image above.
- Select "Unlimited sites (Akismet Business) in the dropdown, verify it replaces the Akismet Pro product in the cart with Akismet Business (of the same billing-term).
- Now select 2, 3, or 4 Sites, and verify Akismet Pro has been added back to the cart in the quantity you selected.
- Change billing-term variation and verify everything still works as intended.
- Also try selecting the number of sites using the keyboard. (`Enter` or `space` opens the dropdown. `up` and `down` arrows to change the quantity, `Enter` to select.)
- Try to break things or find issues.

## Pre-merge Checklist
